<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>指定列</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxColumns">
       <item>
        <property name="text">
         <string>语文</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>数学</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>英语</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>物理</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>分值</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxCompares">
       <item>
        <property name="text">
         <string>==</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>&lt;=</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>&gt;=</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditFindText"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnFind">
       <property name="text">
        <string>查找</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>姓名</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditName"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnAddLines">
       <property name="text">
        <string>增加行</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDelLines">
       <property name="text">
        <string>删除选中行</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDesSelectedItems">
       <property name="text">
        <string>删除选中条目</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
